pkg/openshift/certgen/unstable/templates/master/tmp/ansible/azure-ocp-deploy.yml

```diff
@@ -2,7 +2,11 @@
   hosts: localmaster
   gather_facts: false
 
-  tasks:
+  # we need variables from openshift_facts to be in scope
```

Comment:
@kwoodson why does this file change?

Reply:
@jim-minter, inside of openshift-ansible the template service broker has the 'l_os_registry_url` variable defined.
```
https://github.com/openshift/openshift-ansible/blob/master/roles/template_service_broker/defaults/main.yml#L11
```

This variable is being set in openshift_facts and the expectation is that we include these facts to run these other roles:
https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_facts/defaults/main.yml#L15

I included the facts which brings those variables into scope.

Hope this helps clarify.

Reply:
thanks!